Tests/gui/GSCodingFlags/GSCellFlags.m

```diff
@@ -0,0 +1,62 @@
+#include "Testing.h"
+
+#include <Foundation/NSAutoreleasePool.h>
+#include <Foundation/NSValue.h>
+
+#include <AppKit/NSApplication.h>
+#include "GSCodingFlags.h"
+
+int main()
+{
+    CREATE_AUTORELEASE_POOL(arp);
+    GSCellFlagsUnion mask = { { 0 } };
+
+    START_SET("GSCodingFlags GNUstep CellFlags Union")
+    
+    NS_DURING
```

Comment:
I don't think that we need this check for this test. Nothing that would require a backend is used here.